ludwig/modules/embedding_modules.py

```diff
@@ -137,20 +119,6 @@ def embedding_matrix_on_device(
     )
     if not embeddings_on_cpu and torch.cuda.is_available():
         embeddings.to(device='cuda:0')
-    '''
-    else:
-        embeddings, embedding_size = embedding_matrix(
-            vocab,
-            embedding_size,
-            representation=representation,
-            embeddings_trainable=embeddings_trainable,
-            pretrained_embeddings=pretrained_embeddings,
-            force_embedding_size=force_embedding_size,
-            embedding_initializer=embedding_initializer
-        )
-    '''
```

Comment:
I think we need to force them being on cpu when `embeddings_on_cpu==True` though, otherwise if the model is on gpu also the embeddigns will be on gpu I believe.

Reply:
Done